CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Add `nic` service
 - Use `error.As` for `requestFailed` err
 - `%w` instead of `%s` for some printed errors
+- use `serve` for debug mode
```

Comment:
@cristiGuranIonos not important at all but we should be consistent about whether we use capital letter for the first word of the list or not, maybe we can do that in another PR.

Reply:
done